src/containers/CreateDoc/index.tsx

```diff
@@ -16,11 +19,30 @@ const CreateDoc: React.FC = () => {
     return (
         <React.Fragment>
             <h1>CreateDoc Page</h1>
-            <Dropdown name="documentType" title="Document type" options={documentTypes} />
+            <Dropdown 
+                action={props.setDocType} 
+                name="documentType" 
+                options={documentTypes} 
+                title="Document type" 
+            />
             <button onClick={openPdf}>See preview</button>
             <button onClick={downloadPdf}>Download PDF</button>
+            <p>Selected doc type: {props.docType}</p>
         </React.Fragment>
     );
 }
 
-export default CreateDoc;
\ No newline at end of file
+const mapDispatch = {
+    setDocType: (docType: string) => setDocType(docType)
+}
+
+const mapState = (state: RootState) => ({
+    docType: state.createDoc.documentType
+})
+
+const connector = connect(mapState, mapDispatch);
+
+type PropsFromRedux = ConnectedProps<typeof connector>
```

Comment:
isnt the next line neccesary ? I mean why do we need a `PropsFromRedux` type and then assign to `props` type, the `PropsFromRedux` TYPE ?

Reply:
It was just an example, in this case is not necessary because we dont have extra props, but if we would have some other props, we would do something like this:

interface OwnProps {
  color: string
}

type Props = PropsFromRedux & OwnProps

Anyway, i can remove this line.